polkadot/zombienet_tests/functional/0020-validator-disabling-spam.zndsl

```diff
@@ -0,0 +1,22 @@
+Description: Test validator disabling effects
```

Comment:
please use zombienet-sdk for any new zombienet tests, we should be able to make it less flaky and test more scenarios with confidence

Reply:
I suggest to remove this test for now and implement a proper zombienet-sdk one in a separate PR